### ISSUE TITLE
Loss function used for training

### ISSUE THREAD

**Issue by t0tl** [VicenteVivan/geo-clip#3]
Hello, me and a partner are interested in finetuning the GeoCLIP model however we are unsure of the implementation of the loss function. Could you share the loss function you used or give any tips for implementing it?

**Comment by t0tl** [comment]
In #6 the authors answer this issue. Fyi, you will still have to fix the code yourself as the queue implementation is broken in the repo.

**Comment by VicenteVivan** [comment]
Hi Timothy,

Thank you for your patience and for pointing out the problem. We just updated the repository and corrected the training loop as well as the queue implementation. Apologies for any inconvenience this may have caused. If you have any other questions regarding the implementation please let us know, we would be happy to help.

Sincerely,
Vicente
